main.py

```diff
@@ -1,35 +1,77 @@
+''' Simple aiohttp login system with default user!:
+This system is built with aiohttp module.
+Required Python-3 or 3+
+'''
+
 from aiohttp import web
 
+# Customize the cookie name, value and cookie expire date
+COOKIE_NAME = "OldTamil"
+COOKIE_VALUE = 0
+COOKIE_LIFE_TIME = 24*60*60*30
 
-async def index(request):
-    cookie = request.cookies.get('default', None)
+# User credentials and here we could use more users in future.
+User ='Hippod'
+Password = 'admin'
+
+# Define the required host and port
+HOST = '127.0.0.1'
+PORT = 8080
+
+''' cookie_check function will check for valid cookie on all pages!
+Return True if there is cookie with requested name.
+'''
+def cookie_check(request):
+    cookie = request.cookies.get(COOKIE_NAME, None)
     if cookie:
+        if COOKIE_NAME == COOKIE_NAME:
+            return True
+        # Normal test for checking the COOKIE_NAME
+        else:
+            print("COOKIE_NAME has been modified!.")
+    # Return False when there is no cookie at all.
+    else:
+        return False
+
+# The main page of the website
+async def index(request):
+    Check = cookie_check(request)
+    if Check is True:
         page = open('site.html').read()
```

Comment:
if think it is time to replace this construct: `open('site.html').read()`

What is the problem? Well, you do no error handling at all! What if the file is not available? The program crash! This should never happens!

Please extend and use (somehow, never tried it):

```
def prepare_file(filepath):
    if not os.isfile(filepath):
         print("internal server error, file not found: {}".format(filepath))
         return "<html>INTERNAL SERVER ERROR<html>"
    return open('site.html').read()
```